charts/README.md

```diff
@@ -6,57 +6,92 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
 # Prerequisites
 * [Insall Helm version 3 or later](https://helm.sh/docs/intro/install/)
 
-# Install
-## Install CRDs
-clusterpedia requires CRD resources, which can be installed manually using kubectl, or using the `installCRDs` option when installing the Helm Chart
-> This way references the [cert-manager](https://cert-manager.io/docs/installation/helm/)
+Pull the Clusterpedia repository.
+
+> Currently, the chart has not been uploaded to the public charts repository.
 
-**Option 1: install CRDs with `kubectl`**
 ```bash
-$ kubectl apply -f ./_crds
+git clone https://github.com/clusterpedia-io/clusterpedia.git
+cd clusterpedia/charts
 ```
 
-**Option 2: install CRDs as part of the Helm release**
-To automatically install and manage the CRDs as part of your Helm release, you must add the `--set installCRDs=true` flag to your Helm installation command.
+Since Clusterpedia uses `bitnami/postgresql` and `bitnami/mysql` as subcharts of storage components, it is necessary to add the bitnami repository and update the dependencies of the clusterpedia chart.
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm dependency build
+```
+
+# Choose storage components
+
+The Clusterpedia chart provides two storage components such as `bitnami/postgresql` and `bitnami/mysql` to choose from as sub-charts.
+
+`postgresql` is the default storage component. IF you want to use MySQL, you can add `--set postgresql.enabled=false --set mysql.enabled=true` in the subsequent installation command.
 
-Uncomment the relevant line in the next steps to enable this.
+For specific configuration about storage components, see [bitnami/postgresql](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) and [bitnami/mysql](https://github.com/bitnami/charts/tree/master/bitnami/mysql).
 
+**You can also choose not to install any storage component, but use external components. For related settings, see charts/values.yaml**
```

Comment:
It is already in the charts directory, so you can directly use `[values.yaml](./values.yaml)`